web-client/src/modules/requests/components/RequestList.tsx

```diff
@@ -1,12 +1,13 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Changed RequestList to displaying a list of requestsWIthOffersAndTimeline o a list of requests.  Moved logic for what should happen when you click on an item to RequestItem.  Clicking on an item is a UI action.  Or if we now have a RequestPostItemContainer, the logic could go into the container.  This change I made may be problematic, as I found out later, there is different UI behavior dependin. g on whether the post item is called from MyPinRequestPostsContainer or FindPostsContainer.  However, I think it would be better to have the decision about what is done at the RequestPostItem level, maybe with a flag.